src/components/CustomSelect/CustomSelect.tsx

```diff
@@ -0,0 +1,205 @@
+import { ExpandMore, Check } from '@mui/icons-material';
+import { Select, MenuItem, FormControl, styled, Box, Typography, useTheme } from '@mui/material';
+import React from 'react';
+import type { CustomSelectProps } from './type';
+import type { SelectChangeEvent, Theme } from '@mui/material';
+
+const CustomSelect: React.FC<CustomSelectProps> = ({
+  label,
```

Comment:
What do you think to pass className?:string, for add new styles if its need in others places??

Reply:
in the new update it has been implemented

Reply:
nice, done